recipe/meta.yaml

```diff
@@ -65,13 +66,18 @@ test:
 
 outputs:
   - name: {{ name }}
-    script: {{ go_variant_str }}/build.sh  # [unix]
+    script: {{ go_variant_str }}/build.sh   # [unix]
     script: {{ go_variant_str }}/build.bat  # [win]
+    
+    build:                                    # [linux64 and cgo]
+      binary_relocation: false                # [linux64 and cgo]
+      detect_binary_files_with_prefix: false  # [linux64 and cgo]
+      run_exports:                            # [linux64 and cgo]
+        - sysroot_linux-64                    # [linux64 and cgo]
```

Comment:
You need to use the run_exports from `sysroot_linux-64 2.17`. It is not transitive.

Reply:
Adressed.